app/views/WhatYouOwe.scala.html

```diff
@@ -17,16 +17,17 @@
 @import _root_.implicits.ImplicitDateFormatterImpl
 @import _root_.implicits.ImplicitCurrencyFormatter._
 @import models.financialDetails.WhatYouOweChargesList
-@import java.time.LocalDate
 @import models.financialDetails.DocumentDetailWithDueDate
 @import models.financialDetails._
 @import auth.MtdItUser
 @import views.html.layouts.unifiedLayout
 @import views.html.components._
 @import exceptions.MissingFieldException
 @import uk.gov.hmrc.govukfrontend.views.html.components._
+@import services.DateService
 
 @this(
+    dateService: DateService,
```

Comment:
I would think we don't have to expose services in the views; so on this occasion, this would be more than enough to pass current date which you will get by calling relevant dateService method in the controller